README.md

```diff
@@ -1,45 +1,229 @@
 # DreamHome BackEnd
-
-
+___
 ## About
+- _about_
 
 ## Setup Instructions
-  - how to run the dev/test server locally
-  - how to run production deployed server
+- how to run the dev/test server locally
+- how to run production deployed server
 
 ## Visual Schema
+- ![schema]()
 
 ## Tech stack
+- _stack_
 
 ## Development Database Instructions
+- _instructions_
+
+## Education Endpoints
+- `GET http://localhost:5000/api/v1/education`
```

Comment:
All 'question' attributes of education will have content?

Reply:
I believe that all 'question' fields will have content unless you need it changed?

Reply:
It works better for us if they all have content so its good!